### PR TITLE
[DOCFIX] Update unified namespace docs

### PR DESCRIPTION
**Pull request by gpang**

### CODE REVIEW

**Comment by gpang** [comment]
@ZacBlanco Please take a look.

File: docs/en/core-services/Unified-Namespace.md [hunk 1]
```diff
@@ -68,17 +70,20 @@ Alluxio transparently discovers content present in the underlying storage system
 was not created through Alluxio. For instance, if the underlying storage system contains a directory
 `Data` with files `Reports` and `Sales`, all of which were not created through Alluxio, their
 metadata will be loaded into Alluxio the first time they are accessed, such as when a user requests to
-open a file. The data of the file is not loaded to Alluxio during this process. To load the data into
-Alluxio, one can read the data using `FileInStream` or use the `load` command of the Alluxio shell.
-
+open a file. The contents of the file is not loaded to Alluxio during this process. To load the
+file contents into Alluxio, one can read the data using `FileInStream` or use the `load` command
+of the Alluxio shell.
```
Comment (ZacBlanco) [inline docs/en/core-services/Unified-Namespace.md]: Can you split the sentences in this paragraph into individual lines, similar to some other parts of this doc?

Comment (gpang) [reply docs/en/core-services/Unified-Namespace.md]: Updated

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Comment by gpang** [comment]
alluxio-bot, cherry-pick this to branch-2.2 please

**Pull request merged**